focus.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2023 The TCell Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use file except in compliance with the License.
+// You may obtain a copy of the license at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tcell
+
+// EventFocus is a focus event. It is sent when the terminal window (or tab)
+// gets or loses focus.
+type EventFocus struct {
+	*EventTime
+
+	InOut bool
```

Comment:
Suggestions for a better name welcome...

Reply:
Focused ?

Reply:
OK; I force-pushed with this change.